GATK_QuickByte.md

```diff
@@ -318,13 +359,13 @@ Here is a sample PBS script combining everything we have above, with as much par
 	# Then, we have a number of jobs equal to the number of nodes requested.
 	# Note that MarkDuplicates doesn't take much time, but only uses one core, so it's a bit inefficient here.
 
-	cat $src/sample_list | env_parallel -j $PBS_NUM_NODES --sshloginfile $PBS_NODEFILE \
+	cat $src/sample_list | env_parallel -j 1 --sshloginfile $PBS_NODEFILE \
 		'bwa mem \
-			-t $threads -M \
-			-R "@RG\tID:{}\tPL:ILLUMINA\tLB:{}\tSML{}" \
+			-t 8 -M \
+			-R "@RG\tID:{}\tPL:ILLUMINA\tLB:{}\tSM:{}" \
 			$reference \
-			$src/raw_reads/{}_R1.fastq.gz \
-			$src/raw_reads/{}_R2.fastq.gz \
+			$src/clean_reads/{}_paired_R1.fastq.gz \
```

Comment:
Sorry I had the wrong line. Should be:
$src/clean_reads/{}_paired_1.fastq.gz \

Reply:
Looks like it should be _R1 the way the script is written now, the _1 was only for the raw reads (no major reason why other than that's how they were labeled when downloaded).